Frameworks/UIKit/NSMutableParagraphStyle.mm

```diff
@@ -21,19 +21,32 @@
 @implementation NSMutableParagraphStyle
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (id)copyWithZone:(NSZone*)zone {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self mutableCopyWithZone:zone];
```

Comment:
>mutableCopyWithZone [](start = 17, length = 19)

Check on the reference platform whether `[mutableStyle copy]` gives you an immutable one or a mutable one.

Convention states that a copy of a mutable object is *immutable*, but there are some exceptions.